server/export.go

```diff
@@ -16,10 +16,12 @@ import (
 )
 
 const (
+	DefaultOutputPath    = "./genesis.json"
 	FlagHeight           = "height"
 	FlagForZeroHeight    = "for-zero-height"
 	FlagJailAllowedAddrs = "jail-allowed-addrs"
 	FlagModulesToExport  = "modules-to-export"
+	FlagOutput           = "output"
```

Comment:
```suggestion
	FlagOutput           = "output-document"
```
In the SDK `--output` is used for defining how the output is marshalled (text or json).
We use `--output-document` for exporting to a file.